tests/lib/rules/multiline-comment-style.js

```diff
@@ -1379,6 +1379,25 @@ ${"                   "}
                 { messageId: "missingStar", line: 4 },
                 { messageId: "endNewline", line: 4 }
             ]
+        },
```

Comment:
Can we make this test suite a bit more robust by adding other kinds of formatted comments being formatted to `starred-block`?

Some examples:
```js
////This comment is in
//`separate-lines` format.

// // This comment is in
// `separate-lines` format.


/*
  * // a line comment
 *some.code();
 */


/*
// a line comment
 * some.code();
 */
```

Reply:
The first two are not converted to starred block comments in the current version. So the fix that I am working should not change that behavior right?

Reply:
Oh, interesting. Yes, that makes sense to me. Thanks for adding these tests.